pkg/cover/canonicalizer.go

```diff
@@ -107,22 +118,21 @@ func setModuleKeys(moduleKeys []uint32, modules []host.KernelModule) {
 	sort.Slice(moduleKeys, func(i, j int) bool { return moduleKeys[i] < moduleKeys[j] })
 }
 
+func findModule(pc uint32, moduleKeys []uint32) (moduleIdx int) {
+	moduleIdx, _ = sort.Find(len(moduleKeys), func(moduleIdx int) int {
+		if pc < moduleKeys[moduleIdx] {
+			return -1
+		}
+		return +1
+	})
+	// Sort.Find returns the index above the correct module.
+	return moduleIdx - 1
+}
+
 func convertModulePCs(moduleKeys []uint32, conversionHash map[uint32]*canonicalizerModule, cov []uint32) {
-	// Skip conversion if modules are not used.
```

Comment:
Maybe let it stay to speed up fuzzing when no modules are used?

Reply:
I actually just moved this to the beginning of the Canonicalize/Decanonicalize functions (https://github.com/google/syzkaller/pull/3860/files#diff-ba224a01f923aea6d38679b1cc0e0d6c8315181c7be11973e44cddccd4144f95R92), but if I'm introducing a helper convertPC function I should be able to keep it where it is. I'll work on that now